bin/dotd

```diff
@@ -337,7 +306,7 @@ def main
     end
   end
 
-  do_dtt_subroutine
+  dtt_subroutine
```

Comment:
You've replaced `do_dtt_subroutine` with `dtt_subroutine`, but the method you've introduced is called `dtt_routine`.

Maybe it's time to unit-test this script :grin: